### PR TITLE
removing invalid characters for senate yamls

### PR DESCRIPTION
**Pull request by kandersonea**
https://ngpvan.atlassian.net/browse/VAN-104140

### CODE REVIEW

File: members/B001267.yaml [hunk 1]
```diff
@@ -51,7 +51,7 @@ contact_form:
         - name: Message
           selectors: [ "#field_0F40B55B-8631-4AA2-8223-08DE119CD370" ]
           values: [ "$MESSAGE" ]
-          commands: ["elements[0].value = values[0].replace(/\\b(set forth in the bill)\\b/ig,'').replace(/\\b(is not)\\b/ig,'isn\\'t').replace(/(%2[0C])/g, '').replace(/[(]/g,'[').replace(/[)]/g,']');"]
```
Comment (annanhan) [inline members/B001267.yaml:54]: why did you remove these other pieces?

File: members/D000622.yaml [hunk 2]
```diff
@@ -126,11 +126,11 @@ contact_form:
           value: $MESSAGE
           required: true
     - javascript:
-        - name: char replace
-          values: ["$MESSAGE"]
-          selectors: ["textarea#input-356E5E00-5056-A066-603C-9FAC0F3B6888"]
-          commands: ["for(var i = 0; i < elements.length; i++) { elements[i].value = values[i].replace(/(%2[0C])/g, '').replace(/(CC:)/g,'CC');}"]
```
Comment (annanhan) [inline members/D000622.yaml:132]: why'd you remove the replacement for `CC:` ?

File: members/FAZS4ccc9ed0.yaml [hunk 3]
```diff
@@ -126,7 +126,7 @@ contact_form:
         - name: char replace
           values: ["$MESSAGE"]
           selectors: ["#edit-message"]
-          commands: ["for(var i = 0; i < elements.length; i++) { elements[i].value = values[i].replace(/[#]/g,'');}"]
```
Comment (annanhan) [inline members/FAZS4ccc9ed0.yaml:129]: why did you remove the replacement of `#`?

File: members/FMNS420b9c88.yaml [hunk 4]
```diff
@@ -51,7 +51,7 @@ contact_form:
       - name: char replace
         values: ["$MESSAGE"]
         selectors: ["div textarea[name='message']"]
-        commands: ["for(var i = 0; i < elements.length; i++) { elements[i].value = values[i].replace(/(%2[0C])/g, '').replace(/(CC:)/g,'CC').replace(/[(]/g,'[').replace(/[)]/g,']');}"]
```
Comment (annanhan) [inline members/FMNS420b9c88.yaml:54]: here too

**Review by annanhan — COMMENTED** [review]
i tagged a few places where the old replacements were replaced. there were more spots but i didn't tag them all. why did you replace what was already there instead of adding to the list?

File: members/D000622.yaml (hunk 2 above)
Comment (kandersonea) [reply members/D000622.yaml:132]: this one is fixed

File: members/G000359.yaml [hunk 5]
```diff
@@ -45,7 +45,7 @@ contact_form:
       - name: Message
         selectors: [ "#field_F1684590-B63E-4986-8A57-1216FEFE8B9F" ]
         values: [ "$MESSAGE" ]
-        commands: ["for(var i = 0; i < elements.length; i++) { elements[i].value = values[i].replace(/(%2[0])/g, '');}"]
+        commands: ["for(var i = 0; i < elements.length; i++) { elements[i].value = values[i].replace(/(%2[0C])/g, '').replace(/(;)/g,'').replace(/(—)/g,'').replace(/(-)/g,'');}"]
```
Comment (annanhan) [inline members/G000359.yaml:48]: the original was just `[0]`

**Review by annanhan — APPROVED** [review]
